<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inteDetailDialog</class>
 <widget class="QDialog" name="inteDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>兴趣名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>隶属于：</string>
   </property>
  </widget>
  <widget class="QLabel" name="inteNameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>兴趣名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="inteClassNameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>兴趣名称：</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>专业班级</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择该兴趣的学生：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>116</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addStuButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>批量添加学生(&amp;S)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delStuButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>批量删除学生(&amp;X)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stuDetailButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查看选中学生信息(&amp;C)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
